src/main/java/io/github/railroad/welcome/WelcomeLeftPane.java

```diff
@@ -35,7 +37,7 @@ public WelcomeLeftPane() {
 
         var rightVbox = new RRVBox();
         rightVbox.setAlignment(Pos.CENTER);
-        rightVbox.getChildren().add(new Label("Railroad IDE"));
+        rightVbox.getChildren().add(new LocalizedLabel("railroad.home.window.title"));
```

Comment:
Window title should not be localized
```suggestion
        rightVbox.getChildren().add(new Label("Railroad IDE"));
```